data/assets/scene/solarsystem/missions/dawn/vesta.asset

```diff
@@ -48,10 +48,10 @@ local Vesta = {
       DestinationFrame = "GALACTIC"
     }
   },
+  BoundingSphere = 262000,
```

Comment:
Not related to this PR, really, but since you're changing this you might also just as well set the interaction sphere to the same value so we can't fly into Vesta :)